tests/suites/test_suite_pkcs11_client.data

```diff
@@ -5,3 +5,15 @@ pk_import_sign:"data_files/server1.key"
 PKCS#11 RSA generate and sign
 depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
 pk_generate_sign:MBEDTLS_PK_RSA
+
+PKCS#11 RSA import, sign and verify with Cryptoki
+depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
+pk_import_sign_verify:"data_files/server1.key"
+
+PKCS#11 RSA import, sign with MbedTLS and verify with Cryptoki
+depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
+pk_import_verify_signed:"data_files/server1.key"
```

Comment:
The test descriptions are helpful, but the test function names aren't so much, because they're not indicating which side is doing what. Can we use more self-explanatory names like `pk_rsa_transparent_sign_pkcs11_verify`?